tree/dataframe/inc/ROOT/RDFHelpers.hxx

```diff
@@ -147,6 +148,29 @@ RNode AsRNode(NodeType node)
    return node;
 }
 
+// clang-format off
+/// Trigger the event loop of multiple RDataFrames concurrently
+/// \param[in] handles A vector of RResultHandles
+///
+/// This function triggers the event loop of all computation graphs which relate to the
+/// given RResultHandles. The advantage compared to running the event loop implicitly by accessing the
+/// RResultPtr is that the event loops will run concurrently. Therefore, the overall
+/// computation of all results is generally more efficient.
+/// It should be noted that user-defined operations (e.g., Filters and Defines) of the different RDataFrame graphs are assumed to be safe to call concurrently.
+///
+/// ~~~{.cpp}
+/// ROOT::RDataFrame df1("tree1", "file1.root");
+/// auto r1 = df1.Histo1D("var1");
+///
+/// ROOT::RDataFrame df2("tree2", "file2.root");
+/// auto r2 = df2.Sum("var2");
+///
+/// // RResultPtr -> RResultHandle conversion is automatic
+/// ROOT::RDF::RunGraphs({r1, r2});
+/// ~~~
+// clang-format on
+void RunGraphs(std::vector<RResultHandle> handles);
```

Comment:
Maybe this signature could benefit from a variadic template rather than a vector to input the handles? 
```
template <typename T, typename... Ts>
void RunGraphs(RResultHandle firstgraph, RResultHandle... othergraphs)
```
It would save some copies in passing the vector by value and the interface would become ```ROOT::RDF::RunGraphs(graph1,graph2,graph3)```
At the same time it could make it more difficult to write the implementation code for the function. As you prefer  :smile:

Reply:
The problem is that you want to make book keeping in a complex analysis for which you need a container. Therefore, a signature ala `RunGraphs(p1, p2, ...)` is not very helpful. But indeed, we can add a variadic overload, if we like! However, I don't see yet the requirement.